components/function-controller/hack/webhook_config.yaml

```diff
@@ -8,26 +8,6 @@ reservedEnvs:
 function:
   replicas:
     minValue: "1"
-    defaultPreset: "L"
```

Comment:
You can remove it from webhook chart. (values.yaml file in serverless/charts/webhook)

Reply:
done